components/movementsensor/gpsutils/ntrip.go

```diff
@@ -159,9 +157,11 @@ Loop:
 				}
 				st.Streams = append(st.Streams, str)
 			}
-		case "ENDSOURCETABLE":
-			break Loop
 		default:
+			if strings.Contains(fields[0], "END") {
```

Comment:
If the caster location is "BEND, OREGON", this line might fire when you don't want it to. My instinct is that you're somehow not reading all the data you're supposed to read, and figuring out why that happens and fixing it is likely to be more robust than working around the symptoms. Do you know why you're unable to read to the end of a large sourcetable? If so, I suspect that the answer will suggest a more reliable way to read to the end, and this part of the code doesn't need to change.

Reply:
Could we change `strings.Contains` to `strings.HasPrefix`? That would give me more confidence that we won't have false positives here.